tests/unit/property_value_v2.cpp

```diff
@@ -18,6 +18,21 @@
 #include "storage/v2/temporal.hpp"
 
 using namespace memgraph::storage;
+using enum CoordinateReferenceSystem;
+
+///!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+///!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+///!!!!!!!!!!!THIS FILE IS TOO LONG!!!!!!!!!!!!
+///!!!!!!!!!!!!! TODO: REFACTOR !!!!!!!!!!!!!!!
+///!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+///!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
No, ideally we will eventually have alternative generic variant type which is high performance. Then most of what this test does is redundant. We only have the test ATM because we have to hand code a variant, which is error prone.